src/tribler-gui/tribler_gui/i18n.py

```diff
@@ -0,0 +1,19 @@
+import locale
+
+from PyQt5.QtCore import QCoreApplication, QTranslator
+
+def translate(context, key):
+    return u'%s' % (QCoreApplication.translate(context, key))
+
+def tr(key):
+    return u'%s' % (QCoreApplication.translate("@default", key))
+
+def get_translator_for(locale_id):
+    translator = QTranslator()
+    translator.load(locale_id, "tribler-gui/tribler_gui/i18n")
+    return translator
+
+def get_default_system_translator():
+    defaultLocale = locale.getdefaultlocale()[0]
+    return get_translator_for(defaultLocale)
+    
```

Comment:
Could you remove the trailing spaces from this line*?

*This should've been caught by `pylint` with a `C0303`, but apparently it's broken? 🤔

Reply:
This seems to be an error then in the diff-cover tool, since it triggers when running `pylint` normally.

Reply:
Flake8 could've also caught this I guess, but it is configured for only `--select=E901,E999,F821,F822,F823,I100,I101,I201,I202`.

Reply:
Please use `pre-commit` package to run pre-commit checks and lints automatically. We use the Black formatter with the GUI code.

Reply:
I really tried to read most of the documentation before making the pull request but I completely missed this pre-commit thing. Done that for the files that I have changed now. Should we put this information in a easier to find place?

Reply:
One can say pre-commit was optional and experimental for some time. `contributing.rst` needs to be updated for sure. Also, pre-commit is only used in the GUI, and there are no checks for Black formatting in the PR tester.